src/main.py

```diff
@@ -103,6 +103,63 @@ def login(username):
     c.execute(query,target)
     conn.commit()
 
+def general():
+    choice='*'
+    while choice!='q':
+        print("si- Sign Up\nhc- Help Center\nab- About\npr- Press,\nbl- Blog,\nca- Careers,\nde Developers\nq-quit")
+        choice = input("Please select a option:")
+
+        if choice=='si':
+            createnewacc()
+        if choice=='hc':
+            print("We're here to help")
+        if choice =='ab':
+            print("In College: Welcome to In College, the world's largest college student network with many users in many countries and territories worldwide")
+        if choice=='pr':
+            print("In College Pressroom: Stay on top of the latest news, updates, and reports" )
+        if choice=='bl':
+            print( "Under construction")
+        if choice=='ca':
+            print( "Under construction")
+        if choice=='de':
+            print( "Under construction")
+
```

Comment:
What happens when choice == anything other than the choices listed? Input validation may be beneficial. Also, it may be more beneficial to number the choices instead of two letter input, that is up for debate/preference though